lib/pg_party/hacks/postgresql_database_tasks.rb

```diff
@@ -10,7 +10,14 @@ def run_cmd(cmd, args, action)
 
         partitions = begin
           ActiveRecord::Base.connection.select_values(
-            "SELECT DISTINCT inhrelid::regclass::text FROM pg_inherits"
+            """
+              SELECT
+                inhrelid::regclass::text
+              FROM
+                pg_inherits
+              JOIN pg_class AS p ON inhparent = p.oid
+              WHERE p.relkind = 'p'
+            """
```

Comment:
For consistency can we use heredoc like we do in other places? https://github.com/rkrage/pg_party/blob/562e0835bc75b48f601c00c044a814ba54b2f565/lib/pg_party/adapter_decorator.rb#L94-L98

Reply:
Sure, done.